frb_dart/lib/src/ffi/io.dart

```diff
@@ -51,26 +37,137 @@ class DartApiDl {
 /// This class is only for internal usage.
 class WireSyncReturnStruct extends ffi.Struct {
```

Comment:
Is it possible we also generate this `extends ffi.Struct` info *automatically* via our newly added frb_rust_bindgen_io?

Reply:
Fixed, added memory-checking test and comment.